deploy-manage/upgrade/deployment-or-cluster/saved-object-migrations.md

```diff
@@ -12,41 +12,40 @@ Each time you upgrade {{kib}}, an upgrade migration is performed to ensure that
 ::::
 
 
-::::{warning} 
-{{kib}} 7.12.0 and later uses a new migration process and index naming scheme. Before you upgrade, read the documentation for your version of {{kib}}.
-::::
+% ::::{warning} 
+% {{kib}} 7.12.0 and later uses a new migration process and index naming scheme. % Before you upgrade, read the documentation for your version of {{kib}}.
+% ::::
 
 
 ::::{warning} 
-The `kibana.index` and `xpack.tasks.index` configuration settings are obsolete and no longer taken into account in 8.x. If you are using custom index names, please perform the necessary adaptations before attempting to upgrade to 8.x.
+The `kibana.index` and `xpack.tasks.index` configuration settings are obsolete and no longer considered in 9.x. If you use custom index names, please make the necessary adaptations before upgrading to 9.x.
 ::::
 
 
+## How saved object migrations work [upgrade-migrations-process] 
 
-## How saved objects migrations work [upgrade-migrations-process] 
-
-When you start a new {{kib}} installation, an upgrade migration is performed before starting plugins or serving HTTP traffic. Before you upgrade, shut down old nodes to prevent losing acknowledged writes. To reduce the likelihood of old nodes losing acknowledged writes, {{kib}} 7.12.0 and later adds a write block to the outdated index.
+When you start a new {{kib}} installation, an upgrade migration is performed before starting plugins or serving HTTP traffic. Before you upgrade, shut down old nodes to prevent losing acknowledged writes. To reduce the likelihood of old nodes losing acknowledged writes, {{kib}} adds a write block to the outdated index.
```

Comment:
When you start a new {{kib}} installation, an upgrade migration is performed before starting plugins or serving HTTP traffic. Before you upgrade, shut down old nodes to prevent losing acknowledged writes.
If the upgrade includes breaking changes, the old saved objects will be reindexed into new indices. Otherwise, the existing indices will be reused, and saved object documents will be updated in place.